packages/apps/transfer/.eslintrc.js

```diff
@@ -1,8 +1,32 @@
+const fs = require('fs');
+
+const prettierOptions = JSON.parse(fs.readFileSync('./.prettierrc', 'utf8'));
 require('@rushstack/eslint-config/patch/modern-module-resolution');
 
 module.exports = {
-  plugins: ['jest'],
+  plugins: ['prettier', 'jest'],
   extends: ['next/core-web-vitals', '@kadena-dev/eslint-config/profile/react'],
   parserOptions: { tsconfigRootDir: __dirname },
   ignorePatterns: ['**/*.spec.ts', '**/*.spec.tsx', '**/generated/**'],
+  rules: {
+    'prettier/prettier': ['error', prettierOptions],
+    '@typescript-eslint/strict-boolean-expressions': 0,
+    'arrow-body-style': [2, 'as-needed'],
+    'comma-dangle': [2, 'always-multiline'],
+    'prefer-template': 2,
+    'import/newline-after-import': 1,
+    'import/no-unresolved': 2,
+    'jsx-a11y/aria-props': 2,
```

Comment:
JSX-ally rules are for accessibility 
forcing user to use correct roles in case they used aria-*, also they should be valid regarding the element, for example an anchor IF had aria props, then it should have role of a link